main/server/userWebservices.js

```diff
@@ -97,4 +99,13 @@ module.exports = function (router) {
       next(e)
     })
   })
+
+  router.patch('/users/me',
```

Comment:
We create another route so we do not break eventually existing code depending on `put` version (I did not see one though).